ScoutSuite/frontend/src/components/ServiceStatus/style.scss

```diff
@@ -12,11 +12,19 @@
     color: $success-color;
   }
 
-  &.issues {
+  &.critical {
+    color: $ncc-red;
```

Comment:
I think we should add the variables for the color for each level now. https://ncc-scout.slack.com/files/UDCSGNLQM/F01M7FA2UN8/sc_2021-02-04_18h40m53s.png

Reply:
Done!